config/clients/java/template/client-OpenFgaClientIntegrationTest.java.mustache

```diff
@@ -21,9 +21,17 @@ import org.junit.jupiter.api.BeforeEach;
 import org.junit.jupiter.api.Test;
 import org.junit.jupiter.api.TestInstance;
 import org.junit.jupiter.api.TestInstance.Lifecycle;
+import org.testcontainers.junit.jupiter.Container;
+import org.testcontainers.junit.jupiter.Testcontainers;
+import org.testcontainers.openfga.OpenFGAContainer;
 
 @TestInstance(Lifecycle.PER_CLASS)
+@Testcontainers
 public class OpenFgaClientIntegrationTest {
+
+    @Container
+    private static final OpenFGAContainer openfga = new OpenFGAContainer("openfga/openfga:v1.5.1");
```

Comment:
```suggestion
    private static final OpenFGAContainer openfga = new OpenFGAContainer("openfga/openfga:v1.5.1");
```

Can we move the version to the config? Ideally it should match whatever is in the make file, but that would be N+1